mofacts/client/views/experiment/unitEngine.js

```diff
@@ -1408,8 +1402,7 @@ function scheduleUnitEngine() {
           // 1 - legacy was f/b, now "b" forces a button trial
           // 2 - trial type (t, d, s, m, n, i, f)
           // 3 - location (added to qidx)
-          const groupEntry = group[index * templateSize + k];
```

Comment:
Why is this deleted?

Reply:
I think I misunderstood the point of this line. I didnt realize that a tdf can have multiple groups. Ill change this in a new commit